README.md

```diff
@@ -225,26 +225,35 @@ Optionally pass in an options object as the second argument to `addPlugin` to fu
 const ampPlugin = require('@ampproject/eleventy-plugin-amp');
 module.exports = function (eleventyConfig) {
   eleventyConfig.addPlugin(ampPlugin, {
-    // Disable AMP Cache (enabled by default)
+    // Disable AMP Cache (enabled by default) and self-host the AMP runtime.
```

Comment:
It's not yet valid AMP, but will be in the future.

Reply:
In the future will be be possible to self-host _and_ use the cache? I was thinking it's awkward if a single flag controls both features.

Reply:
Ah got it - yeah that makes sense.

Reply:
But I remember now what my thinking was: `ampCache: false` disables the AMP cache completely which means it also won't use the runtime from cdn.ampproject.org. However, self-hosting is controlled by the `host` option. The name might be too generic though for what it does. Maybe `ampRuntimeHost`?

Reply:
I think it's also possible you might want to disable the AMP cache from search results, but still want to load the runtime from cdn.ampproject.org. Maybe no-one actually wants this in practice, but it's reasonable in theory, and helps keep the concepts separate.

For my personal use case (don't know if this would be useful to anyone else!) I'd like separate controls for:

- Whether search links to my host, or its own separate cache (I guess the third SXG option would need to be configured elsewhere)
- Where the AMP runtime comes from, options:
  - cdn.ampproject.org
    - … LTS version or latest? (default = latest)
  - self-hosted[*]
    - directory on filesystem to put runtime
    - URL path on host (default = "/amp/" or "/rtv/"?)

[*] Does the server hosting the AMP runtime need to match the host of the AMP page itself (i.e. its canonical)? I seem to remember that was a constraint, or potentially so.

Reply:
See https://github.com/ampproject/eleventy-plugin-amp/pull/36